lib/ansible/module_utils/compat/paramiko.py

```diff
@@ -5,10 +5,15 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
+from cryptography.exceptions import InternalError
+
 PARAMIKO_IMPORT_ERR = None
 
 paramiko = None
 try:
     import paramiko
-except (ImportError, AttributeError) as err:  # paramiko and gssapi are incompatible and raise AttributeError not ImportError
+# paramiko and gssapi are incompatible and raise AttributeError not ImportError
+# When running in FIPS mode, cryptography raises InternalError
+# https://bugzilla.redhat.com/show_bug.cgi?id=1778939
+except (ImportError, AttributeError, InternalError) as err:  
```

Comment:
As discussed between me, @jborean93, and @abadger, let's just make this `except Exception as err:`